test/unit/logic/communities.spec.ts

```diff
@@ -344,22 +346,16 @@ describe('Community Component', () => {
 
   describe('when getting public communities', () => {
     const options = { pagination: { limit: 10, offset: 0 }, search: 'test' }
-    const mockCommunities = [
+    const mockCommunities: Omit<CommunityPublicInformation, 'ownerName'>[] = [
```

Comment:
Just to follow our testing standards, non-primitives should be defined inside `beforeEach`s, this makes sure that they're not modified during the execution of the tests and ensures a correct order of definition, has the constants described in the `describe` scopes are executed once before any of the tests are executed.